recipes-bsp/u-boot/u-boot-sunxi.bb

```diff
@@ -22,6 +22,7 @@ SRC_URI = " \
 	git://github.com/linux-sunxi/u-boot-sunxi.git;protocol=git;branch=sunxi \
 	file://0002-gcc5-fixes.patch \
```

Comment:
if you are touching this, can you please rename patches to start from 0001? Thanks

Reply:
Done.